templates/projects/web/project.json

```diff
@@ -10,7 +10,7 @@
     "Microsoft.AspNetCore.Diagnostics": "1.0.0",
     "Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore": "1.0.0",
     "Microsoft.AspNetCore.Identity.EntityFrameworkCore": "1.0.0",
-    "Microsoft.AspNetCore.Mvc": "1.0.0",
```

Comment:
Can you also add an entry for routing see https://github.com/aspnet/Templates/blob/rel/1.0.1/src/BaseTemplates/StarterWeb/project.json#L13